README.md

```diff
@@ -113,6 +113,42 @@ ResponseCode response = SlackWebHookService.with(token)
 RoboSlack handles error responses through the ``ResponseCode`` class. Check your ``ResponseCode`` to see why a message
 may have failed to send.
 
+#### Slack Date Formatting
+
+RoboSlack supports Slack's Date formatting in fields that allow Slack Markdown. 
```

Comment:
I think we should elaborate as to why this is useful, since anyone can just use the java `DateTimeFormatter` on any `Temporal`, why use the `SlackDateTime` API?  What does it grant us?

At a high level, I view the main benefit being that the formatting `SlackDateTime` will be rendered differently based on where and when the client is viewing it.  For example, a `SlackDateTime` message would be rendered as `today` until the viewer then opens the same message the following day to see `yesterday`, where this also will account for the viewers timezone.

These are a few of the things I wanted the docs to convey, apologies if I didn't elaborate this originally.